tests/test_pysoundfile.py

```diff
@@ -6,6 +6,7 @@
 import pytest
 import cffi
 import sys
+import pathlib
```

Comment:
You probably have to catch an `ImportError` here?

Or you move the `import` statement to right where `pathlib` is used?